TurkishFilter/sections/specific.txt

```diff
@@ -41,6 +41,7 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
+illerarasimesafe.com##.adRes
```

Comment:
This rule hides more empty space:
`illerarasimesafe.com##div[class^="adRes"]`
![Screenshot 2021-02-24 at 18 02 29](https://user-images.githubusercontent.com/18536328/109019939-89110f00-76ca-11eb-8cab-f06177fd03e0.png)

Reply:
Fixed.